src/styles/global/helper-classes.scss

```diff
@@ -40,6 +40,22 @@
   }
 }
 
+// Only show when cookies are not supported
+.no-cookies:not(html) {
```

Comment:
I feel that a `cookies` class is too generic; however, we made a change in class names for our most recent theme along these lines already.  To express support for JS in a class that made more sense we switched to: `supports-js` and `supports-no-js`.  

Following the same logic:

```scss
// Only show when browser cookies are not supported
.supports-no-cookies:not(html) {
  display: none;
  
  html.supports-no-cookies & {
    display: block;
  }
}

// Only show when browser cookies are supported
.supports-cookies {
  html.supports-no-cookies & {
    display: none;
  }
}
```